tests/rpm_limits_test.py

```diff
@@ -0,0 +1,34 @@
+import asyncio
+import time
+
+import pytest
+
+from traveltimepy.async_client import AsyncClient
+from traveltimepy.client import Client
+
+
+@pytest.mark.asyncio
+async def test_async_client_10_requests_throttled(async_client_low_rpm: AsyncClient):
```

Comment:
Wondering what guarantee do we get in this test, that it will be a "low rpm" instance passed here?